### DISCUSSION TITLE
API endpoint for adding external responses

### DISCUSSION THREAD

**Discussion by alpkaanaksu** [rubenarslan/formr.org#557] [Q&A] [answered]
Can we send formr results/responses from other software to be included in the data?

**Comment by rubenarslan** [comment, marked as answer]
Yes, people have integrated lab.js or jspsych tasks in formr and stored the data in a hidden text field as JSON. Or if you redirect people to an external platform, you can capture the data either through a get param (if it's not much) or an R API call (if the external platform has an API).
